src/relayFeeCalculator/chain-queries/ethereum.ts

```diff
@@ -0,0 +1,70 @@
+import { QueryInterface } from "../relayFeeCalculator";
+import { BigNumberish } from "../../utils";
+import { Coingecko } from "../../coingecko/Coingecko";
+import ethers from "ethers";
+import axios from "axios";
+
+const { parseUnits } = ethers.utils;
+
+// Note: these are the mainnet addresses for these symbols meant to be used for pricing.
+export const SymbolMapping: { [symbol: string]: { address: string; decimals: number } } = {
```

Comment:
we have a similar mapping in the frontend constants. i wonder if maybe we can override this constant by optionally passing the structure in from the FE, this way we only have to change the constant in one place

Reply:
Done.